src/Illuminate/Support/helpers.php

```diff
@@ -924,16 +924,3 @@ function windows_os()
         return strtolower(substr(PHP_OS, 0, 3)) === 'win';
     }
 }
-
-if (! function_exists('with')) {
```

Comment:
This helper was built to support some internal needs of the core, wasn't even documented for the users. And, after all, is for the next release. Looks reasonable to me to move forward.

I'll wait for more feedbacks about this. Thanks.